model/item.cc

```diff
@@ -39,14 +39,14 @@ namespace minsky
   void BoundingBox::update(const Item& x)
   {
     ecolab::cairo::Surface surf
-       (cairo_recording_surface_create(CAIRO_CONTENT_COLOR_ALPHA,NULL));
+       (cairo_recording_surface_create(CAIRO_CONTENT_COLOR_ALPHA,NULL));    
     auto savedMouseFocus=x.mouseFocus;
     x.mouseFocus=false; // do not mark up icon with tooltips etc, which might invalidate this calc
     x.onResizeHandles=false;
     try
       {
-        x.draw(surf.cairo());
-		cairo_rotate(surf.cairo(),-x.rotation()*M_PI/180);  // perform transformation after drawing, otherwise ink extents not calculated correctly below. For ticket 1232      
+		cairo_rotate(surf.cairo(),x.rotation()*M_PI/180);  // perform transformation after drawing, otherwise ink extents not calculated correctly below. For ticket 1232  
```

Comment:
Firstly, the comment is no longer correct. Secondly, is ticket #1232 still relevant?

Reply:
> Firstly, the comment is no longer correct

I know, that is why it is still only a draft PR.

For the  life of me, I can't understand why the resize arrows can appear correctly on vars and ops (not intop though, but that is why the override exists, so I will have to specialize in its case), work two or three times at any angle, and then stop working.

Whether I use -x.rotation() or +x.rotation() does not seem to make a difference...

> Secondly, is ticket #1232 still relevant?

I can probably fix that using a new Rotate member for the stock variables in GodleyIcon::positionVariables(). 

I might have to move on though, because I already got bogged down on ticket 1264, where I could not even get the needle to move. At least on this one I've been making very slow incremental progress.

Reply:
> For the life of me, I can't understand why the resize arrows can appear correctly on vars and ops (not intop though, but that is why the override exists, so I will have to specialize in its case), work two or three times at any angle, and then stop working.

The bounding box is wrong, so if a var name is long (more than one letter), in any angle other than 0 or 180 degrees, the ports and resize handle can no longer be selected. This in infuriating, because the ports and resize handle appear when the mouse cursor moves over them. I don't understand why the bounding box does not update to the correct shape...

> Secondly, is ticket #1232 still relevant?

I have managed to fix this another way, but the ports of the stocks on Godleys can't be selected if its name is longer than a single letter.

IntOp still a WIP too. Very tricky this one, but I feel like I'm slowly, painfully so, moving towards a better solution than I had previously.

![WorksForVarsWithShortNames](https://user-images.githubusercontent.com/46541934/104636673-02691b00-56a4-11eb-9817-5eac9dea7a35.png)